### ISSUE TITLE
OkHttpSigner throws a NullPointerException when trying to sign GET HTTP requests

### ISSUE THREAD

**Issue by jaaufauvre** [Mastercard/oauth1-signer-java#1]
[OkHttpSigner](https://github.com/Mastercard/oauth1-signer-java/blob/master/src/main/java/com/mastercard/developer/signers/OkHttpSigner.java) throws a NullPointerException when trying to sign GET HTTP requests. 

Indeed, the RequestBody property is null for GET requests and the following test fails:

```
if (builtRequest.body().contentLength() > 0) {
      // ...
}
```

This can be fixed by using:

```
RequestBody body = builtRequest.body();
if (null != body && body.contentLength() > 0) {
     // ...
}
```

**Comment by kkrauth** [comment]
Good catch! Can you please submit a PR?

**Comment by kkrauth** [comment]
Addressed by PR#2, closing.
